content/download/lts.adoc

```diff
@@ -121,8 +121,8 @@ However, if you're using Jenkins 2.47 or Jenkins 2.56, you may have problems dow
 ### Switching Update Sites
 
 The Jenkins project operates multiple update sites that inform Jenkins about available updates to Jenkins and plugins.
-As Jenkins sends its current version when requesting new data, the same URL can be used for all releases of Jenkins, and will always serve the most appropriate update information.
+As Jenkins sends its current version when requesting new data, the same link:http://updates.jenkins-ci.org/update-center.json[URL] can be used for all releases of Jenkins, and will always serve the most appropriate update information.
```

Comment:
the base URL should actually be `https://updates.jenkins.io`, I've fixed this recently in core